src/main/resources/templates/dashboard.html

```diff
@@ -203,4 +199,49 @@
 		</div>
 	</div>
 </div>
+
+<script src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.6.0/Chart.bundle.min.js"></script>
+<script>
+	(function() {
+		window.onload = function() {
+			let ctx = document.getElementById('dashboard-active-hours');
+			let myChart = new Chart(ctx, {
+				type: 'line',
+				data: {
+					labels: ["Wed", "Thu", "Fri", "Sat", "Sun", "Mon", "Tues"],
```

Comment:
Hardcoded from my end to reflect the demo day. Feel free to populate it dynamically.